js_modules/dagit/packages/core/src/assets/__fixtures__/AssetTables.fixtures.ts

```diff
@@ -29,25 +32,25 @@ export const AssetCatalogGroupTableMock: MockedResponse<AssetCatalogGroupTableQu
   },
 };
 
-export const SingleAssetQueryTrafficDashboard: MockedResponse<SingleAssetQuery> = {
+export const SingleAssetQueryTrafficDashboard: MockedResponse<SingleNonSdaAssetQuery> = {
   request: {
-    query: SINGLE_ASSET_QUERY,
+    query: SINGLE_NON_SDA_ASSET_QUERY,
     variables: {input: {path: ['dashboards', 'traffic_dashboard']}},
   },
   result: {
     data: {
       __typename: 'DagitQuery',
       assetOrError: {
```

Comment:
Worked with Marco a bit trying to get these working on Friday but the tldr is that the builders are broken for assets :( 

The reason is that v 3.3 of the builder tool doesn't support union types properly and tries to return a "ConfigType" and not an instance of one of the Config Type. v3.5 fixes that, but causes infinite loops during generation because it follows solid => parent composite solid => solid over and over. Hoping for v3.6 :)